.github/workflows/ruff-check.yml

```
@@ -13,52 +13,66 @@ permissions:
-          # List all changed .py files in target directories
+          
+          echo "BASE_SHA: $BASE_SHA"
+          echo "HEAD_SHA: $HEAD_SHA"
+          
+          # List all changed .py files
           FILES=$(git diff --name-only --diff-filter=ACMRT $BASE_SHA $HEAD_SHA \
                   | grep -E '\.py$' || true)
-
-          echo "files=$FILES" >> "$GITHUB_OUTPUT"
-
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)